po/messages.pot

```diff
@@ -2090,296 +2097,3 @@ msgstr ""
 #: ../Pinta/Pads/ToolBoxPad.cs:42 ../Pinta/Pads/ToolBoxPad.cs:49
 msgid "Tools"
 msgstr ""
-
-#: Polar Inversion dialog
-msgid "Edge Behavior"
-msgstr ""
-
-#: Mono.Addins.Gui/Mono.Addins.Gui/AddinInfoView.cs:123
-msgid ""
-"Multiple selection:\n"
-"\n"
-msgstr ""
-
-#: Mono.Addins.Gui/Mono.Addins.Gui/AddinInfoView.cs:160
```

Comment:
It looks like a lot of strings were removed here, which I wouldn't have expected. Did you generate this by running `make updatepot`?